proxmox/proxmox_instance.go

```diff
@@ -56,16 +57,124 @@ func (p *ProxMox) getNextID() string {
 
 // CreateInstance - Creates instance on Proxmox.
 func (p *ProxMox) CreateInstance(ctx *lepton.Context) error {
+
+	var err error
+
 	config := ctx.Config()
 
 	nextid := p.getNextID()
 
+	instanceName := config.RunConfig.InstanceName
+
+	archType := config.CloudConfig.Arch
+	machineType := config.CloudConfig.Machine
+	socketsNum := config.CloudConfig.Sockets
+	coresNum := config.CloudConfig.Cores
+	numaStr := strconv.FormatBool(config.CloudConfig.Numa)
+	memoryHmn := config.CloudConfig.Memory
 	imageName := config.CloudConfig.ImageName
+	storageName := config.CloudConfig.StorageName
+	isoStorageName := config.CloudConfig.IsoStorageName
+	bridgeName := config.CloudConfig.BridgeName
+	bridgeName0 := config.CloudConfig.BridgeName0
```

Comment:
what's the rationale for 2 of these? the expectation for multiple nics?

Reply:
Yes, because often for production apps need to be have 2 nics, later i want add bridgeName1 for secondary nic. And not only use with DHCP, also will need add a static settings IPs too for secondary nic.

Reply:
so we apparently do support multiple nics, https://github.com/nanovms/nanos/pull/1461 , https://github.com/nanovms/ops/issues/1009 however, ops needs more support to enable this fully, in particular network settings for static assignment need to be added in ops but this should prob be done in a different pr